docs/admin-guide/load-config.md
# The Load Config File

A case and its individuals (or samples, in cancer track) can be uploaded into Scout using specially formatted .yaml config files. Config files contain information about the analysis, panels used, path to VCF and eventual alignment files and more. The following template illustrates the basic structure of the .yaml config file:

The format is [Yaml][yaml].

Example configuration files are found here: `scout/demo/643594.config.yaml` or on [Github](https://raw.githubusercontent.com/Clinical-Genomics/scout/refs/heads/main/scout/demo/643594.config.yaml).

### Deprecation Warnings:
`bam_file`, `bam_path` and `alignment_path` are redundant in internal usage. Future versions of Scout will only
support `alignment_path`.


### Configuration Parameters
Below are available configuration parameters for a Scout case. Names marked with an asterisk (*) are mandatory.

- **analysis_date(*)** _Datetime_ Time for analysis in datetime format. Defaults to time of uploading. Example `2016-10-12 14:00:46`.
- **cnv_report** _String_ Path to the CNV report file.
- **coverage_qc_report** _String_ Path to static coverage and QC report file.
- **cohorts** _List of strings_ Meta organising study participants or cases.
- **collaborators** _List of strings_ List of collaborators.
- **coverage_qc_report** _String_ Path to HTML file with coverage and QC report.
- **custom_images** _Object_ Custom images displayed on case pages or STR variant pages. Supported formats are `gif`, `svg`, `png`, `jpg`, `jpeg`.
    - **case_images** _Dictionary of lists_ Images grouped by section name for case view. Each section will be displayed as a separate panel on the case view. (Alias: `case`)
        - **title** _String_ Image title (mandatory).
        - **description** _String_ Image description (mandatory).
        - **path** _String_ Path to image file (mandatory).
        - **width** _Int_ Optional image width.
        - **height** _Int_ Optional image height.
    - **str_variants_images** _List_ Images connected to STR variants. (Alias: `str`)
        - **title** _String_ Image title (mandatory).
        - **description** _String_ Image description (mandatory).
        - **path** _String_ Path to image file (mandatory).
        - **str_repid** _String_ REPID for the STR variant (mandatory, for example `AFF2`).
        - **width** _Int_ Optional image width.
        - **height** _Int_ Optional image height.
- **default_gene_panels** _List of strings_ List of default gene panels. Variants from the genes in the gene panels specified will be shown when opening the case in scout.
- **delivery_report** _String_: Path to HTML delivery report.
- **exe_ver**: Pipeline detailed software versions (YAML)
- **family(*)**  _String_ Unique ID of the case.
- **family_name**  _String_ Optional name of the case.
- **gene_fusion_report** _String_ Path to a static gene fusion report produced by [Arriba][arriba] containing only clinical fusions (a subset of all detected fusions).
- **gene_fusion_report_research** _String_ Path to a static gene fusion report produced by [Arriba][arriba] containing all detected fusions.
- **gene_panels** _List of strings_ List of gene panels. Specifies what panels the case has been run with.
- **human_genome_build** _String_ Version of genome version used, 37 or 38. Defaults to 37.
- **lims_id** _String_ Case ID in Lims
- **madeline** _String_ Path to a madeline pedigree file in XML format.
- **multiqc** _String_ Path to a [multiqc][multiqc] report with arbitrary information.
- **multiqc_rna** _String_ Path to a [nf-core/rnafusion multiqc][rna-multiqc] report with arbitrary information.
- **omics_files** _List_ List of multiomics results files for the case:
    - **fraser** _String_ Path to TSV file to parse WTS [DROP][drop] FRASER splice outlier omics variants as produced by e.g. [Tomte][tomte]
    - **outrider** _String_ Path to TSV file to parse WTS [DROP][drop] OUTRIDER expression outlier omics variants as produced by e.g. [Tomte][tomte]
    - **methbat** _String_ Path to TSV file to parse LRS [MethBat][methbat] methylation outlier omics variants as produced by e.g. [Nallo][nallo]
- **owner(*)**  _String_ Institute who owns current case. Must refer to existing institute.
- **paraphrase** _String_ Path to a [Paraphrase][paraphrase] json file.
- **peddy_check** _String_ Path to a [peddy][peddy] ped check file.
- **peddy_ped** _String_ Path to a [peddy][peddy] ped file with an analysis of the pedigree based on variant information.
- **peddy_sex** _String_ Path to a [peddy][peddy] ped sex check file.
- **phenotype_terms** _List of strings_ List of phenotype terms.
- **rank_model_url** _String_ Full URL for rank model used when scoring variants. If not defined, server config `RANK_MODEL_LINK_PREFIX` and `_POSTFIX` will be added to make the URL.
- **rank model version** _String_ Which rank model that was used when scoring the variants.
- **rank_score_threshold** _Float_ Only include variants with a rank score above this threshold.
- **reference_info**: Pipeline detailed reference file versions (YAML)
- **RNAfusion_inspector** _String_ Path to HTML [nf-core/rnafusion inspector][rnafusion-inspector] report containing only clinical fusions (a subset of all detected fusions).
- **RNAfusion_inspector_research** _String_ Path to HTML [nf-core/rnafusion inspector][rnafusion-inspector] report containing all detected fusions.
- **RNAfusion_report** _String_ Path to HTML [nf-core/rnafusion report][rnafusion-report] containing only clinical fusions (a subset of all detected fusions).
- **RNAfusion_report_research** _String_ Path to HTML [nf-core/rnafusion report][rnafusion-report] containing all detected fusions.
- **rna_human_genome_build** _String_ Version of reference genome used for RNA components of build. "37" or "38", default "38".
- **rna_delivery_report** _String_: Path to HTML RNA delivery report.
- **saltshaker_report** _String_: Path to HTML Saltshaker MT analysis report.
- **samples** _List_ List of samples included in the case:
    - **alignment_path** _String_ Path to BAM/CRAM file to view alignments.
    - **analysis_type** _String_ Specifies the analysis type for the sample. Options: {wgs, wgs-lr, wes, wts, panel, panel-lr, panel-umi, ogm, unknown, external}.
    - **assembly_alignment_path** _String_ Path to BAM/CRAM file to view de novo assembly alignments.
    - **bam_file** _String_ Path to BAM/CRAM file to view alignments **WARNING:** Soon to be deprecated, use *alignment_path*.
    - **bam_path** _String_ Path to BAM/CRAM file to view alignments **WARNING:** Soon to be deprecated, use *alignment_path*.
    - **capture_kit** _String_ Exome specifies the capture kit.
    - **chromograph_images** _List_
        - **autozygous** _String_ Path to file.
        - **coverage** _String_ Path to file.
        - **upd_regions** _String_ Path to file.
        - **upd_sites** _String_ Path to file.
    - **confirmed_parent** _Bool_ True if parent confirmed.
    - **d4_path** _String_ Path to [.d4 file][d4_file]. Required for Chanjo2 integration
    - **father** _String/Int_ Sample ID for father or 0.
    - **hrd** _Int_ Homologous recombination deficiency.
    - **is_sma** _Bool/None_ if SMA status determined - None if not done.
    - **is_sma_carrier**  _Bool/None_  # True / False if SMA carriership determined - None if not done.
    - **minor_allele_frequency_wig** _String_ Path to minor allele frequency bigwig [Reference][hificnv].
    - **mitodel** _String_ Path to mitodel file.
    - **mother** _String/Int_ Sample ID for mother or 0.
    - **msi** _Int_ Microsatellite instability [0-60].
    - **mt_bam** _String_ Path to the reduced mitochondrial BAM/CRAM alignment file.
    - **paraphase_alignment_path** _String_ Path to BAM/CRAM file to view Paraphase alignments [Reference][paraphase].
    - **phase_blocks** _String_ Path to GTF file to view phase blocks.
    - **phenotype(*)** _String_ Specifies the affection status {affected, unaffected, unknown}.
    - **reviewer** _List_ [Reference][srs]
      - **alignment** _String_ Path to BAM/CRAM file to view STR alignments
      - **alignment_index** _String_ Path to BAM/CRAM index file to view STR alignments
      - **trgt** _Bool_ Boolean indicating if individual has trgt data (indicates alignment is spanning, catalog is BED)
      - **vcf** _String_ Path to STR VCF file to view STR alignments
      - **catalog** _String_ Path or URL to REViewer catalog JSON or BED file to view STR alignments
      - **reference** _String_ Path or URL for REViewer to reference sequence for the individual STR alignment
    - **rna_alignment_path** _String_ Path to RNA alignment file (BAM/CRAM)
    - **rna_coverage_bigwig** _String_ Path to coverage islands file generated
    - **omics_sample_id** _String_ Sample ID for RNA, as in outliers files
    - **rhocall_bed** _String_ Path to BED file to view alignments [Reference][rhocall].
    - **rhocall_wig** _String_ Path to WIG file to view alignments [Reference][rhocall].
    - **sample_id(*)** _String_ Identifier for a sample.
    - **sample_name**: _String_ Name of sample.
    - **sex (*)** _String_ One of: {male, female, unknown}. Sex of the sample in human readable format.
    - **smn1_cn** _Int_ Copy number.
    - **smn2_cn** _Int_ Copy number.
    - **smn2delta78_cn** _Int_ Copy number.
    - **splice_junctions_bed** _String_ Path to indexed junctions .bed.gz file
    - **subject_id** _String_ Individual identifier - multiple samples could belong to the same individual
    - **tiddit_coverage_wig** or **coverage_wig** _String_ Path to WIG file to view alignment coverage overview from e.g. [Reference][tiddit] or [Reference][hificnv].
    - **tissue_type** _String_ Sample tissue origin i.e. blood, muscle.
    - **tmb** _Int_ Tumor mutational burden [0, 1000] (tumor case only).
    - **tumor_purity** _Float_ Purity of tumor sample [0.1, 1.0] (tumor case only).
    - **tumor_type** _String_ Type of tumor (tumor case only).
    - **upd_regions_bed** _String_ Path to BED file to view alignments [Reference][upd].
    - **upd_sites_bed** _String_ Path to BED file to view alignments [Reference][upd].
    - **vcf2cytosure** _String_ Path to CGH file to allow download per individual. Such SV files can be visualized using standard arrayCGH analysis tools. See [vcf2cytosure](https://github.com/NBISweden/vcf2cytosure/blob/master/README.md).
- **smn_tsv** _String_ Path to an [SMNCopyNumberCaller][smncopynumbercaller] TSV file
- **somalier_ancestry** _String_ Path to a [Somalier][somalier] ancestry tsv file.
- **somalier_pairs** _String_ Path to a [Somalier][somalier] pairs tsv file.
- **somalier_samples** _String_ Path to a [Somalier][somalier] samples tsv file.
- **status** _String_ Any of the following strings: 'prioritized', 'inactive', 'ignored', 'active', 'solved' or 'archived'.
- **synopsis** _String_ Synopsis of case.
- **sv_rank_model_version** _String_ Rank model that was used when scoring the variants.
- **track** _String_ Type of track: {"rare", "cancer"}. Default: "rare".
- **sv_rank_model_url** _String_ Full URL for SV rank model used when scoring SV variants. If not defined, server config `SV_RANK_MODEL_LINK_PREFIX` and `_POSTFIX` will be added to make the URL.
- **sv_rank_model_version** _String_ SV rank model version used when scoring SV variants.
- **vcf_cancer** _String_ Path to cancer VCF file (tumor case only).
- **vcf_cancer_research** _String_ Path to VCF file with all variants (tumor case only).
- **vcf_snv** _String_ Path to SNV VCF file  containing only clinical variants (a subset of all variants).
- **vcf_snv_research** _String_ Path to VCF file with all variants.
- **vcf_sv** _String_ Path to SV VCF file containing only clinical variants (a subset of all variants).
- **vcf_sv_research** _String_ Path to VCF file with all SV variants.


See [loading case](loading-case.md) for more information on how to load a case using the config file, as well as details on
custom images and custom reports.

### Example Minimal config

Here is an example of a minimal load config:

```yaml
---

owner: cust004

family: '1'
samples:
  - analysis_type: wes
    sample_id: NA12878
    capture_kit: Agilent_SureSelectCRE.V1
    father: 0
    mother: 0
    sample_name: NA12878
    phenotype: affected
    sex: male

vcf_snv: scout/demo/643594.clinical.vcf.gz
```


[arriba]: https://arriba.readthedocs.io/en/latest/
[d4_file]: https://github.com/38/d4-format
[drop]: https://github.com/gagneurlab/drop
[hificnv]: https://github.com/PacificBiosciences/HiFiCNV
[methbat]: https://github.com/PacificBiosciences/MethBat
[multiqc]: https://github.com/ewels/multiqc
[nallo]: https://github.com/genomic-medicine-sweden/nallo
[paraphase]: https://github.com/PacificBiosciences/paraphase
[paraphrase]: https://github.com/Clinical-Genomics/paraphrase
[peddy]: https://github.com/brentp/peddy
[rna-multiqc]: https://nf-co.re/rnafusion/output#multiqc
[rnafusion-inspector]: https://nf-co.re/rnafusion/output#fusioninspector
[rnafusion-report]: https://nf-co.re/rnafusion/output#fusion-report
[rhocall]: https://github.com/dnil/rhocall
[smncopynumbercaller]: https://github.com/Illumina/SMNCopyNumberCaller
[somalier]: https://github.com/brentp/somalier
[srs]: https://github.com/Clinical-Genomics/Scout-REViewer-service
[tiddit]: https://github.com/SciLifeLab/TIDDIT
[tomte]: https://github.com/genomic-medicine-sweden/tomte
[upd]: https://github.com/bjhall/upd
[yaml]: https://yaml.org




docs/admin-guide/igv-settings.md
## IGV browser settings

### Default tracks

The default file formats for viewing alignments in Scout is [BAM][bam] or [CRAM][cram]. Scout utilizes the embeddable interactive genome visualization tool [igv.js][igv] to display sample tracks and the following default tracks, available in the IGV browser by default for both genome builds GRCh37 (hg19) and GRCh38 (hg38):

- Reference genome track
- Genes track
- ClinVar SNVs track
- ClinVar CNVs track

Reference genome and genes track are collected from the [Broad Institute Amazon S3 storage][amazon_s3], while the ClinVar tracks are available in the UCSC Genomics Institute [Sequence and Annotation Downloads][ucsc_downloads]


### Custom tracks

One or more custom tracks could be additionally loaded and visualized in the Scout IGV browser by including them in the general Scout configuration file.
To add a public track of [type "variant"][igv_variant_track] to Scout use the following example line:
```
CLOUD_IGV_TRACKS = [
    {
        "name": "custom_public_bucket",
        "access": "public",
        "tracks": [
            {
                "name": "Display name of track",
                "type": "variant",
                "format": "vcf",
                "build": "37", # or "38"
                "url": "url_to_resource",
                "indexURL": "url_to_resource_index",
            }
        ],
    },
]
```


[bam]: https://software.broadinstitute.org/software/igv/BAM
[cram]: https://software.broadinstitute.org/software/igv/FileFormats/CRAM
[igv]: https://github.com/igvteam/igv.js
[amazon_s3]: https://s3.amazonaws.com/igv.broadinstitute.org
[ucsc_downloads]: https://hgdownload.soe.ucsc.edu/downloads.html
[igv_variant_track]: https://github.com/igvteam/igv.js/wiki/Variant-Track


CHANGELOG.md
# Change Log
All notable changes to this project will be documented in this file.
This project adheres to [Semantic Versioning](http://semver.org/).

About changelog [here](https://keepachangelog.com/en/1.0.0/)

## [unreleased]
### Changed
- Replaced Ensembl rest liftover service with liftover API from the Broad Institute (#6293)

## [4.111.3]
### Fixed
- Reintroduce GRCh38 alias conversion for export panels (#6289)

## [4.111.2]
### Fixed
- Partly revert alias change, and instead use GRCh38 to mean write chr prefix in export commands (#6288)

## [4.111.1]
### Fixed
- Build version alias GRCh38 to 38 for consistency in export commands (#6283)

## [4.111.0]
### Added
- Custom MT Saltshaker HTML report loading (#6242)
- Command to download all PanelApp panels to file (#6239)
- Alternatively load and update PANELAPP-GREEN panel from a pre-downloaded file (#6244)
- Command to remove loaded rank models (#6262)
### Changed
- Default gnomAD linkout for hg38 to non_uk_biobank subset (#6250)
- Handle conflicts and duplicates when uploading managed variants (#6256, #6258, #6259)
- On caseS page, dynamically hide empty categories of cases (#6260)
- `scout download everything` command downloads Ensembl data last, so that if the Ensembl download fails, it does not prevent the other files from being downloaded (#6269)
### Fixed
- Saltshaker report update command (#6246)
- Managed variants counter, wrongly defaulting to build 37 (#6252)
- Managed variants matching follows build on case page (#6254)
- Updated docs on load case images (#6267)
- Blog posts for 4.111 and 4.110 releases, removed obsolete `expected_coverage` key from documentation (#6270)
- LoqusDB setting reset by non-admin changes to institute page (#6275)
- CLI mt report command to use both M and MT chr names. Update documentation on MT var loading. (#6277)

## [4.110.0]
### Added
- A documentation page for institute settings (#6212)
- Parse and display Paraphrase output from Nallo Paraphase conversion (#6178)
- Command line option to export causative variants by category and/or genome build (#6202)
- An additional button on causatives and verified pages to download SNVs and SVs as input for the managed variants list (admins only) (#6205 and #6231)
- Display total number of variants and number of variants returned by a filter on Managed Variants page (#6223)
### Changed
- Genome build is now shown on variant verification "Sanger" emails (#6194)
- Refactor, speedup (dry-run only) and add a progress bar to `scout delete variants` cmd (#6094)
- `scout delete variants` command now accepts an optional `--out-file` where to print a detailed report of the deletion process (#6094)
- Gens fallback to v3 if no GENS_VERSION given, and no version detected from Gens API (#6195)
- Refactor variant formatting logic (pretty_variant) out of Jinja2 template (#6218)
### Fixed
- Formatting of a list on managed variant export documentation (#6203)
- Exception for singletons from the "Include variants present only in unaffected" filter (#6209)
- Reverted STR exception to show unaffected: the singleton exception for unaffected is enough (#6197, #6234)
- Link to Alamut Plus software on Institute Settings page (#6213)
- Give custom Jinja2 filters a separate app module (#6220)
- Case page image accordion style (#6230)
- Width of comment field on order Sanger form (#6229)
- Links from Paraphrase page to IGV (#6235)

## [4.109.3]
### Fixed
- Revert form data passing for SNVs (#6191)

## [4.109.2]
### Fixed
- Fix more variant page `compounds follow filter` issues (#6188)

## [4.109.1]
### Fixed
- Clinical filter application for SV, MEI etc. (#6181)
- Fix variants page crashing when selecting `compounds follow filter` checkbox (#6184 and #6187)
- Fix scout export managed variants (#6186)

## [4.109]
### Added
- Add a `--type` to the export case CLI, to limit the exported case search by analysis type (#6089)
- `MANE Transcripts` as a default IGV track for alignments in genome build 38 (#6127)
- Accept single end BNDs (#6132)
- Update analysis type for case's individual/sample using `scout update individual` command (#6142)
- Filter methbat outliers by significance ("HypoMethylated", "HyperMethylated", "HypoASM", "HyperASM") (#6140)
- `--force` flag to force load research variants hitting a certain gene/region into a clinical case (#6153)
- `Download as PDF` button on ACMG sandbox page (#6152)
- Links to gDNA IGV view from variantS page also for WTS outliers (#6151)
- Pick phenotype terms (HPO, OMIM, ORPHA) to be assigned to a case from a phenomodel object (#6138)
- Methylation outliers population frequency filter and clinical filter (#6156)
### Changed
- Export genotype with gene variantS search (#6088)
- igv.js version to 3.8.0 - fixes `e.tags` error popup issue (#6096, #6118)
- Rework rank model file loading, allowing full URI (#6090)
- Refactored `scout delete` commands and relative tests into separate modules (#6102)
- The gens extension now generates version appropriate links (#6105)
- Refactored snv and cancer snv variants code to share same routines as other variant categories (#6116 and #6128)
- On managed variants page, print maintainers names instead of emails/ids (#6121)
- Enable ClinVar test API URL by default on Scout demo config settings (#6129)
- ClinVar settings (API and list of users allowed to send API submissions) are no longer shown only to admin users (#6134)
- Refactor `load research` CLI to raise a FileNotFoundError with aggregated missing configured research files per case (#6148)
- Make gene panel's display name editable by users, if they are panel maintainers or admins (#6147)
- Rank model URLs can now be filenames, if not starting with "http" (#6150)
- Methbat outliers significance filtering extended with "AlleleSpecificMethylation" and "imprint" (#6156)
- Custom IGV tracks settings shown on variant don't depend on the specific variant genome build (#6165)
### Fixed
- Comments' text wrapping in ACMG classifications exported as PDF (#6086)

scout/server/config.py
# -*- coding: utf-8 -*-
SECRET_KEY = "this is not secret..."
REMEMBER_COOKIE_NAME = "scout_remember_me"  # Prevent session timeout when user closes browser
# INSTANCE_NAME = "Development" # Name will be displayed on the top navigation menu
# INSTANCE_COLOR = "#800000" # Color of the top navigation menu

# SESSION_TIMEOUT_MINUTES = 60  # Minutes of inactivity before session times out

# MONGO_URI = "mongodb://127.0.0.1:27011,127.0.0.1:27012,127.0.0.1:27013/?replicaSet=rs0&readPreference=primary"
MONGO_DBNAME = "scout"

BOOTSTRAP_SERVE_LOCAL = True
TEMPLATES_AUTO_RELOAD = True

DEBUG_TB_INTERCEPT_REDIRECTS = False

# Flask-mail: http://pythonhosted.org/flask-mail/
# see: https://bitbucket.org/danjac/flask-mail/issue/3
MAIL_SERVER = "smtp.gmail.com"
MAIL_PORT = 587
MAIL_USE_TLS = True
MAIL_USE_SSL = False

# Filename of accrediation bagde image in server/bluprints/public/static
# If null no badge is displayed in scout
ACCREDITATION_BADGE = "swedac-1926-iso17025.png"

# LDAP login Settings
# Complete list of accepted parameters available here: https://github.com/rroemhild/flask-ldapconn
# LDAP_HOST = "localhost" # Can also be named LDAP_SERVER
# LDAP_PORT = 389
# LDAP_BASE_DN = 'cn=admin,dc=example,dc=com # Can also be named LDAP_BINDDN
# LDAP_USER_LOGIN_ATTR = "mail" # Can also be named LDAP_SEARCH_ATTR
# LDAP_USE_SSL = False
# LDAP_USE_TLS = True

# Parameters required for Google Oauth 2.0 login
# GOOGLE = dict(
#    client_id="client.apps.googleusercontent.com",
#    client_secret="secret",
#    discovery_url="https://accounts.google.com/.well-known/openid-configuration",
# )

# Parameters required for login with Keycloak
# KEYCLOAK = dict(
#    client_id="<name_of_client>",
#    client_secret="secret",
#    discovery_url="http://localhost:8080/realms/<name_of_realm>/.well-known/openid-configuration",
#    logout_url="http://localhost:8080/realms/<name_of_realm>/protocol/openid-connect/logout",
# )

# Chanjo database connection string - used by chanjo report to create coverage reports
# SQLALCHEMY_DATABASE_URI = "mysql+pymysql://test_user:test_passwordw@127.0.0.1:3306/chanjo"

# URL to an instance of Chanjo2, for generating coverage report
# CHANJO2_URL = "http://chanjo2-stage.scilifelab.se"

# Configure gens service
# GENS_HOST = "127.0.0.1"
# GENS_PORT = 5000
# GENS_VERSION = 4

# Connection details for communicating with a rerunner service
# RERUNNER_API_ENTRYPOINT = "http://rerunner:5001/v1.0/rerun"
# RERUNNER_TIMEOUT = 10
# RERUNNER_API_KEY = "I am the Keymaster of Gozer"

# Matchmaker connection parameters
# - Tested with PatientMatcher (https://github.com/Clinical-Genomics/patientMatcher) -
# MME_ACCEPTS = "application/vnd.ga4gh.matchmaker.v1.0+json"
# MME_URL = "http://localhost:9020"
# MME_TOKEN = "DEMO"

# Beacon connection settings
# - Tested with cgbeacon2 (https://github.com/Clinical-Genomics/cgbeacon2) -
# BEACON_URL = "http://localhost:6000/apiv1.0"
# BEACON_TOKEN = "DEMO"

# ClinVar API URL
# Comment the line below to use the official (non-test) ClinVar API
# Documentation: https://www.ncbi.nlm.nih.gov/clinvar/docs/api_http/
CLINVAR_API_URL = "https://submit.ncbi.nlm.nih.gov/apitest/v1/submissions/"

# connection details for LoqusDB MongoDB database
# Example with 2 instances of LoqusDB, one using a binary file and one instance connected via REST API
# When multiple instances are available, admin users can modify which one is in use for a given institute from the admin settings page
# LOQUSDB_SETTINGS = {
#    "default" : {"binary_path": "/miniconda3/envs/loqus2.5/bin/loqusdb", "config_path": "/home/user/settings/loqus_default.yaml"},
#    "loqus_api" : {"api_url": "http://127.0.0.1:9000"},
# }

# Connection details for Scout REViewer service
# SCOUT_REVIEWER_URL = "http://127.0.0.1:8000/reviewer"

# Custom IGV tracks can be configured here to allow users to enable them on their IGV views.
# A number of publicly-available tracks can be found here: https://trackhubregistry.org/ or downloaded from
# CUSTOM_IGV_TRACKS = [
#    {
#        "name": "public_tracks",
#        "access": "public",
#        "tracks": [
#            {
#                "name": "dbVar common SVs (global)",
#                "type": "annotation",
#                "format": "bigbed",
#                "build": "37",
#                "url": "https://ftp.ncbi.nlm.nih.gov/pub/dbVar/sandbox/dbvarhubtest/hg19/common_global.bb",
#            },
#            {
#                "name": "dbVar common SVs (global)",
#                "type": "annotation",
#                "format": "bigbed",
#                "build": "38",
#                "url": "https://ftp.ncbi.nlm.nih.gov/pub/dbVar/sandbox/dbvarhubtest/hg38/common_global.bb",
#            },
#        ],
#    },
#    {
#        "name": "local_tracks",
#        "access": "public",
#        "tracks": [
#            {
#                "name": "MANE",
#                "type": "annotation",
#                "format": "bigbed",
#                "build": "38",
#                "url": "scout/resources/custom_igv_tracks/mane.bb",
#            },
#        ],
#    },
# ]

# Chanjo-Report
REPORT_LANGUAGE = "en"
ACCEPT_LANGUAGES = ["en", "sv"]

# FEATURE FLAGS
SHOW_CAUSATIVES = True
SHOW_OBSERVED_VARIANT_ARCHIVE = True
HIDE_ALAMUT_LINK = True

# URL of a general web page where users can place orders for analyses or reruns
RERUN_URL = "https://clinical.scilifelab.se/"
# Display case rerun monitoring toggle
RERUN_MONITOR = True

# OMIM API KEY: Required for downloading definitions from OMIM (https://www.omim.org/api)
# OMIM_API_KEY = 'valid_omim_api_key'

# Parameters for enabling Phenomizer queries
# PHENOMIZER_USERNAME = "test_user"
# PHENOMIZER_PASSWORD = "test_password"

# Parameters for Phenopacket-api backend
PHENOPACKET_API_URL = "https://phenopacket-stage.scilifelab.se"

# Rank model link
RANK_MODEL_LINK_PREFIX = "https://raw.githubusercontent.com/Clinical-Genomics/reference-files/master/rare-disease/rank_model/rank_model_-v"
RANK_MODEL_LINK_POSTFIX = "-.ini"
SV_RANK_MODEL_LINK_PREFIX = "https://raw.githubusercontent.com/Clinical-Genomics/reference-files/master/rare-disease/rank_model/svrank_model_-v"
SV_RANK_MODEL_LINK_POSTFIX = "-.ini"

# BioNano Access Server extension
# BIONANO_ACCESS = "https://bionano-access.scilifelab.se"
# BIONANO_USERNAME = "USERNAME"
# BIONANO_PASSWORD = "PASSWORD"

# Uncomment and customise to log users' activity to file
# USERS_ACTIVITY_LOG_PATH = "users_activity.log"
